smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/HttpBindingProtocolGenerator.java

```diff
@@ -214,6 +216,72 @@ public void generateResponseSerializers(GenerationContext context) {
         }
     }
 
+    @Override
+    public void generateRouter(GenerationContext context) {
+        TopDownIndex topDownIndex = TopDownIndex.of(context.getModel());
+
+        context.getWriter().addImport("restjson", null, "@aws-smithy/server-common");
```

Comment:
This class isn't specific to json - it also is implemented by rest xml for example. I don't see anything below that's json specific, so this should just be renamed.

Reply:
Makes sense.